tests/installed/destructive/staged-deploy.yml

```diff
@@ -4,8 +4,18 @@
 - name: Write staged-deploy commit
   shell: |
     ostree --repo=/ostree/repo commit --parent="${commit}" -b staged-deploy --tree=ref="${commit}" --no-bindings
+    orig_mtime=$(stat -c '%.Y' /sysroot/ostree/deploy)
     ostree admin deploy --stage staged-deploy
+    new_mtime=$(stat -c '%.Y' /sysroot/ostree/deploy)
+    assert_not_streq "${orig_mtime}" "${new_mtime}"
     test -f /run/ostree/staged-deployment
+    ostree refs | grep -E -e '^ostree/' | while read ref; do
+      if test "$(ostree rev-parse ${ref})" = "${commit}"; then
```

Comment:
Shouldn't we be looking for the checksum of the new commit we just created here? (I.e. `ostree rev-parse staged-deploy`).

Reply:
Spotting tests that subtly successfully test the wrong thing is :medal_sports: 

Fixup :arrow_down: